src/app/api/models/user/user.service.spec.ts

```diff
@@ -89,7 +96,7 @@ describe('UserService', () => {
     tick();
   }));
 
-  it('should delete a user', fakeAsync(() => {
+  xit('should delete a user', fakeAsync(() => {
```

Comment:
Is `xit` intentional?

Reply:
Yes, that's intentional. We can remove the test altogether if needed.